0xMycaleum #zkThon Solution #1

0xMycaleum opened the pull request:
## Description

This pull request is for the zkEVM Challenge 1. I have reviewed the zkEVM documentation and shared my understanding through a Twitter thread using the hashtags #zkThon and #zkEVM. The link to my Twitter thread is [https://twitter.com/Mycaleum/status/1639099287304028160].

## Checklist

- [X] I have created a new file named `solution-1.md` for this challenge.
- [X] I have added the details of my submission to the Airtable form [https://airtable.com/shr21z0FfPImZfYBQ].
- [X] I have named my branch `username_zkthon`.

## Additional Information

Upward and onwards frens.

PriyathamVarma reviewed (changes requested):
Review: Thanks for sharing about zkEVM. But as part of the challenge you need to tweet about your understanding of zkEVM. We checked at 12:42 pm GMT on 24/03/2023. We are assuming you might be tweeting the other posts. So will keep the PR on review for now. 
Thank you

0xMycaleum commented:
Sorry for the delayed correction and thank you for the guidance. Is the PR
still currently open for review? I have made a new thread, resubmitted the
AirTable details and updated the Solution 1.md file. Please let me know if
there is anything else I need to do to qualify. Thank you fren!

New thread link is here:
https://twitter.com/Mycaleum/status/1640125056146890757
Corrected solution MD file is here:
https://github.com/0xMycaleum/zkThon-2023-is-here-baby-/blob/0xMycaleum_zkThon/solution-1.md

On Fri, Mar 24, 2023 at 6:44 AM Vatsavaye priyatham Varma <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Review: Thanks for sharing about zkEVM. But as part of the challenge you
> need to tweet about your understanding of zkEVM. We checked at 12:42 pm GMT
> on 24/03/2023. We are assuming you might be tweeting the other posts. So
> will keep the PR on review for now.
> Thank you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Polygon-Advocates/zkThon-2023/pull/35#pullrequestreview-1356624504>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATFX3DTAF53LEQEAUF3VKHTW5WJKHANCNFSM6AAAAAAWGAV57U>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
No signature required

0xMycaleum commented:
I do believe I have completed all three challenges. Thank you for your patience and the aavesome intro to the incoming zkEVM launch.
